README.md

```diff
@@ -31,6 +31,18 @@
   </a>
 </p>
 
+<p align="center">
+  <a href="https://pennylane.ai/carnival/">
+    <img width=100% src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/carnival-callout/doc/_static/carnival_splash_bg.png">

```

Comment:
```suggestion
    <img width=100% src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/carnival_splash_bg.png">
```

Reply:
yep I had that on my list, was just verifying that it was working